Sources/PitchSpeller/Data Structures/DirectedEdge.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DirectedEdge.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 7/1/18.
+//
+
```

Comment:
As with previous PR, can you add doc comments? These structures are very elegantly wired up, and I think the docs can help show that.

Reply:
Is there a good place to put more top-level comments that describe relationships?

Reply:
If proper types are put in code back-ticks (`Pitch`, `Frequency`), they should become links within the generated docs.

We can put together more top-level places for architectural explanation. We've had versions of that in the past, and would be worth reviving.